core/ustring.cpp

```diff
@@ -699,7 +699,7 @@ bool String::operator<(const String &p_str) const {
 	return operator<(p_str.get_data());
 }
 
-signed char String::nocasecmp_to(const String &p_str) const {
+int32_t String::nocasecmp_to(const String &p_str) const {
```

Comment:
Did we ever choose to upgrade vectors to be int64?

Reply:
Vectors currently use `int32_t` and I don't think that will change in the near future. The limits of `int32_t` are pretty high for most use cases, so using or allowing to use `int64_t` is a lower priority than, for example, doubles.